src/Element/CivicrmSelectOptions.php

```diff
@@ -50,13 +50,23 @@ public static function valueCallback(&$element, $input, FormStateInterface $form
     }
   }
 
-  protected static function getFieldOptions($form_key) {
+  protected static function getFieldOptions($form_key, $webform_id = NULL) {
     \Drupal::getContainer()->get('civicrm')->initialize();
     $field_options = \Drupal::service('webform_civicrm.field_options');
-    return $field_options->get(['form_key' => $form_key], 'create', []);
+    $data = [];
+    if (!empty($webform_id)) {
+      $webform = \Drupal::entityTypeManager()->getStorage('webform')->load($webform_id);
+      $handler_collection = $webform->getHandlers('webform_civicrm');
+      $handler = $handler_collection->get('webform_civicrm');
+      $settings = $handler->getConfiguration()['settings'];
+      $data = $settings['data'];
+    }
+    return $field_options->get(['form_key' => $form_key], 'create', $data);
```

Comment:
Main fix in PR is Passing missing `$data` element ,  if we look at D7 code : https://github.com/colemanw/webform_civicrm/blob/7.x-5.x/includes/wf_crm_admin_form.inc#L107
```
  private function initializeForm() {
    $this->settings = isset($this->node->webform_civicrm) ? $this->node->webform_civicrm : $this->defaultSettings();
    $this->data = &$this->settings['data'];
  }
```
same `$this->data` variable used everywhere to populate option value for any field.

We don't have that initialisation in  D8 in these options files , handler object is missing, What we doing is here , getting webform id and pull data from settings and pass it  `get` to populate the option as per form key.